freqtrade/tests/strategy/test_strategy.py

```diff
@@ -380,7 +379,7 @@ def test_deprecate_populate_indicators(result):
     with warnings.catch_warnings(record=True) as w:
         # Cause all warnings to always be triggered.
         warnings.simplefilter("always")
-        resolver.strategy.advise_buy(indicators, 'ETH/BTC')
+        resolver.strategy.advise_buy(indicators, {'pair': 'ETH/BTC'})
```

Comment:
this was a typo, and wasn't relevant as in this cases we use the legacy strategy, which does not pass pair.